micro-benchmarks/DRB175-barrier-orig-no.c

```diff
@@ -0,0 +1,35 @@
+/*
+!!!~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~!!!
+!!! Copyright (c) 2017-20, Lawrence Livermore National Security, LLC
+!!! and DataRaceBench project contributors. See the DataRaceBench/COPYRIGHT file for details.
+!!!
+!!! SPDX-License-Identifier: (BSD-3-Clause)
+!!!~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~!!!
+ */
+/**
+ * IS.C: This file is part of kernel of the NAS Parallel Benchmarks 3.0 IS suit.
+ * Intel Inspector can not correctly analysis the master, and report a false postive.
```

Comment:
It looks you mix two patterns into one file.  Can you have two separated files ? One with master , the other with barrier.